### PR TITLE
Update locales to use convars and update `.github`

### PR DESCRIPTION
**Pull request by TonybynMp4**
Updated the locales to use the convar,
updated .github to the one of the core. (prob needs an update)

## Checklist

- [x] I have personally loaded this code into an updated qbcore project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team in the discord at https://discord.com/invite/qbcore. All
```
Comment (Manason) [inline .github/CODE_OF_CONDUCT.md]: old link

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+name: Bug report
+description: Create a report to help us improve or fix something
+title: "[BUG]"
+labels: bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out a bug report!
+        Please use our Discord Server to ask questions and receive support: https://discord.gg/qbcore
```
Comment (Manason) [inline .github/ISSUE_TEMPLATE/bug_report.yml]: old link

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: QBCore Discord Server
+    url: https://discord.gg/qbcore
```
Comment (Manason) [inline .github/ISSUE_TEMPLATE/config.yml]: old

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+name: Feature request
+description: Suggest an idea for QBCore
+title: "[SUGGESTION]"
+labels: enhancement
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please use our Discord Server to ask questions and receive support: https://discord.gg/qbcore
```
Comment (Manason) [inline .github/ISSUE_TEMPLATE/feature_request.yml]: link

**Review by Manason — CHANGES_REQUESTED** [review]

File: fxmanifest.lua [hunk 5]
```diff
@@ -11,7 +11,7 @@ shared_scripts {
 
     'config.lua',
     '@qb-core/shared/locale.lua',
-    'locales/en.lua', -- Change to the language you want
+    'locales/*.lua',
```
Comment (D4isDAVID) [inline fxmanifest.lua:15]:
```suggestion
    'locales/en.lua',
    'locales/*.lua',
```
`locales/en.lua` is also needed here.

File: locales/fr.lua [hunk 6]
```diff
@@ -0,0 +1,22 @@
+local Translations = {
+    error = {
+        invalid_ext = "Ce n'est pas une extension valide, seul",
+        allowed_ext = "extensions sont autorisés.",
+
```
Comment (D4isDAVID) [inline locales/fr.lua]:
```suggestion
        invalid_ext = "Ce n'est pas une extension valide, seul",
```
`allowed_ext` is not used.

File: .github/auto_assign.yml [hunk 7]
```diff
@@ -5,13 +5,13 @@ addReviewers: true
...
 
 # A list of reviewers to be added to pull requests (GitHub user name)
-reviewers:
-  - /maintenance
-
-# A list of keywords to be skipped the process that add reviewers if pull requests include it
+reviewers: 
+  - QBCore-Remastered/maintenance
+  
+# A list of keywords to be skipped the process that add reviewers if pull requests include it
```
Comment (D4isDAVID) [inline .github/auto_assign.yml]:
```suggestion
reviewers:
  - QBCore-Remastered/maintenance

# A list of keywords to be skipped the process that add reviewers if pull requests include it
```
Remove trailing spaces

File: .github/auto_assign.yml [hunk 8]
```diff
@@ -5,13 +5,13 @@ addReviewers: true
 addAssignees: author
 
 # A list of reviewers to be added to pull requests (GitHub user name)
-reviewers:
-  - /maintenance
-
-# A list of keywords to be skipped the process that add reviewers if pull requests include it
+reviewers: 
+  - QBCore-Remastered/maintenance
```
Comment (D4isDAVID) [inline .github/auto_assign.yml]:
```suggestion
  - Qbox-project/maintenance
```

File: .github/workflows/lint.yml [hunk 9]
```diff
@@ -13,11 +13,11 @@ jobs:
         with:
           capture: "junit.xml"
           args: "-t --formatter JUnit"
-          extra_libs: mysql+polyzone+qblocales
+          extra_libs: mysql
```
Comment (D4isDAVID) [inline .github/workflows/lint.yml]:
```suggestion
          extra_libs: mysql+qblocales
```
The linter should know that we use locales.

**Review by D4isDAVID — CHANGES_REQUESTED** [review]

File: fxmanifest.lua (hunk 5 above)
Comment (TonybynMp4) [reply fxmanifest.lua:15]: that's useless tho we're already calling it 🤔

Comment (D4isDAVID) [reply fxmanifest.lua:15]: The other locales use English as their fallback language, so we have to ensure that it loads first.

File: locales/fr.lua [hunk 10]
```diff
@@ -0,0 +1,20 @@
+local Translations = {
+    error = {
+        invalid_ext = "Ce n'est pas une extension valide, seul",
```
Comment (D4isDAVID) [inline locales/fr.lua]: I don't think you finished this sentence?

Comment (YvanCywan) [reply locales/fr.lua]: It's not finished, 
```suggestion
        invalid_ext = "Ce n'est pas une extension valide, seul les liens d'extension %{fileext} sont autorisés",
```

Comment (TonybynMp4) [reply locales/fr.lua]: > 

yup forgor to update this one

File: locales/fr.lua [hunk 11]
```diff
@@ -1,6 +1,6 @@
 local Translations = {
     error = {
-        invalid_ext = "Ce n'est pas une extension valide, seul",
+        invalid_ext = "Ce n'est pas une extension valide, seul les liens utilisant les extensions {fileext} sont autorisées. ",
```
Comment (YvanCywan) [inline locales/fr.lua]: Just missing a %
```suggestion
        invalid_ext = "Ce n'est pas une extension valide, seul les liens utilisant les extensions %{fileext} sont autorisées. ",
```

**Review by YvanCywan — CHANGES_REQUESTED** [review]

File: locales/fr.lua [hunk 12]
```diff
@@ -0,0 +1,20 @@
+local Translations = {
+    error = {
+        invalid_ext = "Ce n'est pas une extension valide, seul les liens utilisant les extensions %{fileext} sont autorisées. ",
```
Comment (D4isDAVID) [inline locales/fr.lua]:
```suggestion
        invalid_ext = "Ce n'est pas une extension valide, seul les liens utilisant les extensions %{fileext} sont autorisées.",
```
Remove trailing space.

**Review by D4isDAVID — CHANGES_REQUESTED** [review]

**Review by D4isDAVID — APPROVED** [review]
Nice

**Review by YvanCywan — APPROVED** [review]

**Review by Manason — APPROVED** [review]

**Comment by TonybynMp4** [comment]
@ChatDisabled 🔫

File: .github/ISSUE_TEMPLATE/config.yml [hunk 13]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: QBox Discord Server
```
Comment (D4isDAVID) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
  - name: Qbox Discord Server
```
Should have a lowercase `b` just like everywhere else.

**Review by D4isDAVID — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md [hunk 14]
```diff
@@ -1,10 +1,11 @@
...
 
-**Questions (please complete the following information):**
-- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest)
-- Does your code fit the style guidelines? [yes/no]
-- Does your PR fit the contribution guidelines? [yes/no]
+## Checklist
+
+<!-- Put an x inside the [ ] to check an item, like so: [x] -->
+
+- [ ] I have personally loaded this code into an updated qbcore project and checked all of its functionality.
```
Comment (D4isDAVID) [inline .github/pull_request_template.md]:
```suggestion
- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 15]
```diff
@@ -0,0 +1,37 @@
+name: Feature request
+description: Suggest an idea for QBox
```
Comment (D4isDAVID) [inline .github/ISSUE_TEMPLATE/feature_request.yml]:
```suggestion
description: Suggest an idea for Qbox
```

**Review by D4isDAVID — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 13 above)
Comment (ChatDisabled) [reply .github/ISSUE_TEMPLATE/config.yml]: qb-lapraces also has that :o

Comment (D4isDAVID) [reply .github/ISSUE_TEMPLATE/config.yml]: Absolute bozo

Comment (TonybynMp4) [reply .github/ISSUE_TEMPLATE/config.yml]: i copied the thing from the core so y'all are bozos :)

Comment (D4isDAVID) [reply .github/ISSUE_TEMPLATE/config.yml]: Incorrect, core still says `QBCore`

Comment (TonybynMp4) [reply .github/ISSUE_TEMPLATE/config.yml]: > Incorrect, core still says `QBCore`

even worse then lol, anyway i copied the one from the core thinking you'd have updated it, turned out it didn't you kindly pointed out (and almost everytime made the changes yourself 👍🏻 ) all of them but i didn't change the other PRs i had at the same time so i guess ima update all of the repos with the right .github now :)

Comment (D4isDAVID) [reply .github/ISSUE_TEMPLATE/config.yml]: Btw when I said bozo I was referring to @ChatDisabled since he is the one that merged it 🤨

Comment (TonybynMp4) [reply .github/ISSUE_TEMPLATE/config.yml]: oh yeah i got that but i kinda took it too cauz afterall im the one writting that lol

**Pull request merged**